pkg/cmd/attestation/trustedroot/trustedroot.go

```diff
@@ -0,0 +1,134 @@
+package trustedroot
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cli/cli/v2/pkg/cmd/attestation/auth"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/io"
+	"github.com/cli/cli/v2/pkg/cmd/attestation/verification"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/sigstore/sigstore-go/pkg/tuf"
+	"github.com/spf13/cobra"
+)
+
+type Options struct {
+	Public      bool
+	Private     bool
+	TufUrl      string
+	TufRootPath string
+	Logger      *io.Handler
+}
+
+type tufClientInstantiator func(o *tuf.Options) (*tuf.Client, error)
+
+func NewTrustedRootCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command {
+	opts := &Options{}
+	trustedRootCmd := cobra.Command{
+		Use:   "trusted-root [--public | --private | --tuf-url <url> --tuf-root <file-path>]",
+		Args:  cobra.ExactArgs(0),
+		Short: "Get a trusted_root.json file, likely for offline verification",
+		Long: heredoc.Docf(`
+			### NOTE: This feature is currently in beta, and subject to change.
+
+            Get a trusted_root.json file, likely for offline verification.
```

Comment:
One thing that I don't think is clear with this command versus the (now removed) `tuf-root-verify` command is the intent. The `tuf-root-verify` command is about ensuring that the published tuf repository is not compromised, i.e by providing the `root.json` oob any client can verify this. While that is still possible with this command, I don't think it's obvious, and if if fails, would it be clear why? I think this can be solved with added documentation to this comand.